custom_components/flair/cover.py

```diff
@@ -0,0 +1,133 @@
+import logging
+from homeassistant.util.percentage import ordered_list_item_to_percentage
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.components.cover import (
+    CoverEntity,
+    SUPPORT_SET_TILT_POSITION,
+    DEVICE_CLASS_DAMPER,
+    SUPPORT_OPEN_TILT,
+    SUPPORT_CLOSE_TILT,
+)
+from .const import (
+    DOMAIN,
+    VENT_CLOSED,
+    VENT_HALF_OPEN,
+    VENT_OPEN
+)
+
+ORDERED_NAMED_VENT_STATES = [VENT_HALF_OPEN, VENT_OPEN]
+
+VENT_STATE_TO_HASS = {
+    VENT_CLOSED: 0,
+    VENT_HALF_OPEN: ordered_list_item_to_percentage(ORDERED_NAMED_VENT_STATES, VENT_HALF_OPEN),
+    VENT_OPEN: ordered_list_item_to_percentage(ORDERED_NAMED_VENT_STATES, VENT_OPEN)
+}
+
+HASS_FAN_SPEED_TO_FLAIR = {v: k for (k, v) in VENT_STATE_TO_HASS.items()}
```

Comment:
Don't need any of this as the Cover entity doesn't have `speed count` like the fan entity